helm/operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: A Helm chart for MinIO Operator
 name: operator
-version: 4.5.2
-appVersion: v4.5.2
+version: 4.5.3
+appVersion: v4.5.3
```

Comment:
As I know, you should not update operator or chart versions because it happens during release process.

Reply:
Fixed